handler/routes.go

```diff
@@ -5,4 +5,5 @@ import "net/http"
 func Routes() {
 	http.Handle("/ping", PingHandler())
 	http.Handle("/login", LoginHandler(deps.userService))
+	http.Handle("/assets", ListAssetHandler(deps.assetService))
```

Comment:
Rename the func as ListAssetsHandler. 
Please check if asset handler file is also pushed here.